package.json

```diff
@@ -8,6 +8,7 @@
   "license": "GPL-3.0",
   "dependencies": {
     "irc": "^0.5.2",
+    "jsonfile": "^3.0.0",
```

Comment:
We can use the inbuilt `fs` module instead of using of adding an extra dependency isn't it?